linera-service/src/linera/client_options.rs

```diff
@@ -4,6 +4,7 @@
 use std::{env, iter, num::NonZeroU16, path::PathBuf, time::Duration};
 
 use chrono::{DateTime, Utc};
+use futures::FutureExt;
```

Comment:
Nit: I generally prefer to not introduce the name if it isn't used (`use futures::FutureExt as _;`).